### PR TITLE
Contactinfo

### LINKED ISSUE #5: Add Contact section
Generally, on a portfolio website, we have the option to contact the person directly using the website form. 
So, it will be great if we can add that section and perhaps link the form response to the email.

image

It can be something along the line of this.

### PR DESCRIPTION
**Pull request by shivamsouravjha**
What I've done:-
- I've added a contact info form where the viewer can enter the name, mail, message and send an email to contact us.
Why have I done:
- In order to enable viewers to have conversations with the portfolio creators.
## Please, go through these steps before you submit a PR.

- [X] You've explained what the Pull Request is adding.
- [X] You've explained why this is important.

Closes #5 

CC @hkaur008 for review/suggestion.

### CODE REVIEW

**Comment by shivamsouravjha** [comment]
I'm trying to resolve the errors.

**Review by hkaur008 — APPROVED** [review]

**Comment by hkaur008** [comment]
@shivamsouravjha  are these resolved ?

File: _sass/contact.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+.contact {
+	background-color: var(--secondaryColor);
```
Comment (rosekamallove) [inline _sass/contact.scss]: You should use the variables present in `_sass/theme.scss` and add more variables if needer.

File: _sass/all.scss [hunk 2]
```diff
@@ -85,7 +85,12 @@ a {
         font-size: medium;
         color: #404040;
     }
-}
+    
+    .username {
+        font-size: medium;
+        color: #404040;
```
Comment (rosekamallove) [inline _sass/all.scss]: Here too

File: _sass/contact.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	border-bottom:1px solid var(--borderColor);
+	overflow:auto;
+}
+
+.card-contact {
+    display: block;
+    flex-direction: row;
+    align-items: center;
+    margin: 0 auto;
+    background-color: var(--mainColor);
```
Comment (rosekamallove) [inline _sass/contact.scss]: This background color doesn't seem to look good, again _use already created variables_ this just disappears with the background.

**Comment by shivamsouravjha** [comment]
@hkaur008 yes resolved!

**Review by rosekamallove — CHANGES_REQUESTED** [review]
Screenshot from 2021-07-22 22-20-23

- The card just disappears with the background and is not in line with the rest of the.
- What you can do is @KUHOO-S had the idea, you can just make it blend with the background.
- Maybe use `bootstrap classes` to style the `input fields` and `submit buttons`

File: _sass/contact.scss (hunk 3 above)
Comment (shivamsouravjha) [reply _sass/contact.scss]: That's some cool idea,let me update it.

**Comment by shivamsouravjha** [comment]
@rosekamallove Idk why I'm getting this bug ,any clue about this? 
Screenshot (276)

**Comment by rosekamallove** [comment]
Yeah, you don't really need to worry about it if it's not causing any problem in the functionality of your PR. I fixed it already in my PR yesterday. (Basically, @hkaur008 suggested I change the file location so I did it and fixed the src)

**Comment by shivamsouravjha** [comment]
yeah yeah, I remember the suggestion thing, cool if it's fixed then.

**Comment by awilkescom** [comment]
Hey, whats the expected behaviour for this? 

For me, it opens my email client and creates an example message

Put it buts the name and email in the content of the message.

How does the portfolio author change the email to be sent to ?

I think we just need to document the feature a bit better so it's clear what to expect from this change

image

**Comment by shivamsouravjha** [comment]
@awilkescom so this feature is a way for the visitor to contact the portfolio owner, and yes this is the desired behaviour of the email. I will comment on the area from where we can change the email.

**Comment by awilkescom** [comment]
Ok, thanks!

**Pull request merged**